pipeline/restart_node.sh

```diff
@@ -591,6 +603,7 @@ do
 			run_with_retries fetch_binaries || break
 		fi
 		run_with_retries kill_harmony || break
+		run_with_retries cleanup_exp_db || break
```

Comment:
only need to run it on explorer node, no need to run it on every node, too slow.